hack/make-rules/test-e2e-node.sh

```diff
@@ -95,6 +95,7 @@ if [ "${remote}" = true ] ; then
   cleanup=${CLEANUP:-"true"}
   delete_instances=${DELETE_INSTANCES:-"false"}
   preemptible_instances=${PREEMPTIBLE_INSTANCES:-"false"}
+  test_timeout=${TIMEOUT:-"45m"}
```

Comment:
Should we avoid hardcoding "45m" here (ref. https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/remote/remote.go#L34), and only use the `test-timeout` flag when the timeout is given? And if so, should we then still write `Defaults to 45m` in the Makefile?

Reply:
Yeah, I'm plus one on avoiding hardcoding 45m - lets keep `remote.go` as the source of truth for the default. I think the `Makefile` should also indicate that `remote.go` contains the source of truth.